android/app/build.gradle

```diff
@@ -1,5 +1,7 @@
 apply plugin: 'com.android.application'
 
+def dropboxAppID = ""
```

Comment:
Thanks for automating the manifest modification process ! That's really cool we won't have to do that by hand.

I have two small things for improvement here:
1. the `resValue` is currently called twice for debug and release build types even though it carries the same value. It can be done this way in one place:
https://gist.github.com/paweldomas/beeef390489e82542f2e7f713846b538#file-build-gradle-L31
2. I'm not sure if declaring the `dropboxAppID` here is the cleanest way. It seems that we never use 'def' in the build file this way, so I'd propose to put it in the android/gradle.properties file like this:
https://gist.github.com/paweldomas/beeef390489e82542f2e7f713846b538#file-gradle-properties-L24
It seems that if the value is defined in the build file directly then it can not be overridden from the properties file.